org.plomgrading.PlomClient.json

```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "org.plomgrading.PlomClient",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "PlomClient",
+    "finish-args": [
+        "--socket=x11",
+        "--share=network"
+    ],
+    "modules": [
+        {
```

Comment:
nitpick: we usually put the dependencies before the module that builds the application itself